mitodl/bootcamp-ecommerce
# Configured IdP for NovoEd to enable login via SAML

#### What are the relevant tickets?
Closes #1008 

#### What's this PR do?
Configure bootcamps as a SAML Identity Provider for NovoEd's Service Provider to allow single sign-on to NovoEd when a user logs in on bootcamps

#### How should this be manually tested?
This can be tested in the PR build: https://bootcamp-ecommerce-ci-pr-1015.herokuapp.com/

- Create a user
- Enroll that user in the `test-bootcamp` course using a Django shell (`novoed.api.enroll_in_novoed_course` - use `test-bootcamp` for the novoed stub value)
- Visit the SP-initiated login URL: https://app.novoed.com/saml/sso?provider=mitbootcamps

This should redirect you to bootcamps, then instantly redirect you to NovoEd since you're already logged in. You should see a dashboard when you land back on NovoEd's app with the `test-bootcamp` course displayed

Also try this in an incognito window to make sure that the redirection works when you are not already logged in
